src/routes/Chat/container.js

```diff
@@ -127,7 +127,7 @@ export class ChatContainer extends Component {
 				Livechat.sendMessage({ msg, token, rid }),
 			]);
 		} catch (error) {
-			const reason = error?.data?.error ?? error.message;
+			const reason = (error && error.data && error.data.error.error) || error.message;
```

Comment:
Why was this change needed? 👀

Reply:
I was getting an error of `Unexpected token` while running `yarn run I18n` command. It seems it's not liking the typescript usage here :(

Reply:
Please check the PR description for error logs